fbchat/models.py

```diff
@@ -267,10 +267,20 @@ def __init__(self, url=None, size=None, name=None, is_malicious=None, **kwargs):
         self.name = name
         self.is_malicious = is_malicious
 
-class AudioAttachment(FileAttachment):
-    def __init__(self, **kwargs):
-        """Represents an audio file that has been sent as a Facebook attachment - *Currently Incomplete!*"""
-        super(StickerAttachment, self).__init__(**kwargs)
+class AudioAttachment(Attachment):
+    # Name of the file
+    filename = None
+    # playable_url
+    playable_url = None
+    # duration
```

Comment:
Maybe you could elaborate on the documentation comments? Something like `Duration of the audioclip in milliseconds`